src/components/RegistrationProblem/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, memo } from 'react'
+import { Link } from 'react-router-dom'
+import InputField from '../InputField'
+import TextArea from '../TextArea'
+import ButtonContainer from '../ButtonContainer'
+import './styles.scss'
+
+const RegistrationProblem = () => {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [message, setMessage] = useState('')
+  return (
+    <main className="registration-problem-container">
+      <h1 className="registration-problem-main-title">Kohdataan</h1>
```

Comment:
The title is a little bit too wide. That might change when the font changes though

Reply:
It did :)